content/en/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/index.md

```diff
@@ -0,0 +1,190 @@
+
+## Telemetry API: Tracing Overview
+
+The Telemetry API offers tracing behavior configuration control over the following at the mesh, namespace, and workload levels:
+
+- **provider selection** - allows selection of backend providers
+([providers](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider)
+are configured in `MeshConfig`) for reporting.
+
+- **sampling percentage** - allows control the rate of trace sampling applied to received requests *for which no prior sampling decision has been made*.
+
+- **custom tags** - allows control over any custom tags to add to each generated tracing span.
+
+- **tracing participation** - allows opting services out of reporting spans to the selected tracing providers.
+
+### Workload Selection
+
+Individual workloads within a namespace are selected via a [`selector`](/docs/reference/config/type/workload-selector/#WorkloadSelector)
+which allows label-based selection of workloads.
+
+It is not valid to have two different `Telemetry` resources select the same workload using `selector`. Likewise, it is not valid to have two
+distinct `Telemetry` resources in a namespace with no `selector` specified.
+
+### Scope, Inheritance, and Overrides
+
+Telemetry API resources inherit configuration from parent resources in the Istio configuration hierarchy:
+
+1.  root configuration namespace (example: `istio-system`)
+1.  local namespace (namespace-scoped resource with **no** workload `selector`)
+1.  workload (namespace-scoped resource with a workload `selector`)
+
+A Telemetry API resource in the root configuration namespace, typically `istio-system`, provides mesh-wide defaults for behavior.
+Do not specify any workload-specific selector in the root configuration namespace (it will be ignored).
+
+Namespace-specific overrides for the mesh-wide configuration can be achieved by applying a new `Telemetry` resource in the desired
+namespace (without a workload selector). Any `Tracing` fields specified in the namespace configuration will completely override
+the field from the parent configuration (in the root configuration namespace).
+
+Workload-specific overrides can be achieved by applying a new Telemetry resource in the desired namespace *with a workload selector*.
+Any `Tracing` fields specified in the namespace configuration will completely override the field from any parent configuration
+(root configuration or local namespace).
+
+## Using the Telemetry API for Tracing configuration
+
+### Configuring tracing providers
+
+Tracing providers are the backend collectors and processors that receive tracing spans
+and process them for storage and retrieval. Example providers include Zipkin, Jaeger, Lightstep,
+Datadog, and Apache SkyWalking.
+
+For Istio, tracing providers are configured for use within the mesh via `MeshConfig`. To configure new
+providers to use in tracing, edit the `MeshConfig` for your mesh via:
+
+{{< text bash >}}
+$ kubectl -n istio-system edit configmap istio
```

Comment:
Not a big fan of edit cm workflow. Should we use "Operator" or Helm YAML here?

Reply:
We can, when that level of support is provided. It doesn't exist at the moment.